manifests/router/connector.pp

```diff
@@ -25,12 +27,29 @@
   Optional[String] $sasl_mech = 'ANONYMOUS',
   Optional[String] $sasl_username = undef,
   Optional[String] $sasl_password = undef,
+  Stdlib::Absolutepath $sasl_password_file = '/etc/qpid-dispatch/connector_pw.conf',
```

Comment:
How about including the title here:
```suggestion
  Stdlib::Absolutepath $sasl_password_file = '/etc/qpid-dispatch/connector-${name}-pw.conf',
```

Reply:
Though not 100% sure about `$name` vs `$title` here. Perhaps you have to use `$title` but the unit tests should make it quite easy to figure out.